src/data/recipes.js

```diff
@@ -392,3 +392,34 @@ export const goldIngot = itemify({
     state.purchasedSmelter && state.itemsSold[items.goldOre.id] >= 50,
   recipeType: recipeType.FORGE,
 })
+
+export const compost = itemify({
```

Comment:
The purpose of this item may not be obvious to players, so we might want to add a `description` explaining what it's for.